docs/quickstart/readme.md

```diff
@@ -0,0 +1,310 @@
+</details>
+
+Type in `Player` and click on `my_game::Player`.
+
+> [!TIP]
+> If this does not show up, you have forgotten to call `register_type` with it.
+
+Now click `Add`. You can now set all the player's fields in a neat Blender UI. Note that while we are only using `f32` here, you can use any kind of field, even complicated nested enums or structs.
+
+<details>
+<summary>The player component in Blender</summary>
+<img src="img/player_component.png" width = 50%/>
+</details>
+
+Congratulations, you have just created your first blueprint.
+
+## Compose the world
+
+Let's populate our world now. Switch back to the `World` scene in Blender as described before.
+
+We will add the following kinds of objects to our world:
+
+- Some basic ground, which we will create directly in the world
+- A camera
+- A light
+- The player character, which will be a blueprint instance
+
+First, let's add a camera through `Add` -> `Camera`. Blender will make the camera face whatever direction the viewport is aimed at, which will probably be completely arbitrary.
+Move the camera a bit so that it faces the world origin and is rotated the right way.
+Pressing `0` on your numpad will show you a preview of what the camera sees.
+You can just copy our transform if you like:
+
+- Location:
+  - X: `0`
+  - Y: `-10`
+  - Z: `2`
+- Rotation
+  - X: `80`
+  - Y: `0`
+  - Z: `0`
+- Scale
+  - X: `1`
+  - Y: `1`
+  - Z: `1`
+
+  All transforms we are using assume that you are using the `XYZ Euler` mode for rotation.
+
+<details>
+<summary>The camera transform in Blender</summary>
+<img src="img/camera_transform.png" width = 50%/>
+</details>
+
+Now, let's add a ground with `Add` -> `Mesh` -> `Cube`. Again scale and move it so that it lies neatly under the camera. Our transform looks like this:
+
+- Location:
+  - X: `0`
+  - Y: `0`
+  - Z: `0`
+- Rotation
+  - X: `0`
+  - Y: `0`
+  - Z: `0`
+- Scale
+  - X: `5`
+  - Y: `5`
+  - Z: `0.1`
+
+Add a directional light with `Add` -> `Light` -> `Sun`.
+This light's position and scale do not matter, only its rotation is actually used. We are using the following:
+
+- Rotation
+  - X: `20`
+  - Y: `20`
+  - Z: `0`
+
+Finally, it is time to create an instance of our player blueprint! For this, go to `Add` -> `Collection Instance` and select `Player`. It will spawn at origin, which means directly inside the ground for us. Move it a bit up. A location of `Z: 1.1` should placed it just a tiny bit above the ground.
+
+And with that, our Blender scene is finished! The final result should look like this:
+
+<details>
+<summary>The final Blender world scene</summary>
+<img src="img/final_blender.png" width = 100%/>
+</details>
+
+Save your Blender file again. This will update the `World.glb` file in the `assets` directory and create a new file for our player blueprint under `assets/blueprints/Player.glb`, including again a metadata file.
+
+<details>
+<summary>The new blueprint files</summary>
```

Comment:
should this not be moved to the "create a blueprint" section instead ?

Reply:
Nope, since that would require marking the `Player` as an `Asset` as it is not used in the `World`. I don't want to add that info as otherwise, a newcomer might think they need to mark every blueprint as an asset.

Reply:
Ahh yes good point!